transform/models/marts/repo_pushes_daily.sql

```diff
@@ -0,0 +1,9 @@
+select distinct
+  repo_id,
+  repo_name,
+  date_trunc('day', event_date) as date_day,
+  count(*) as push_count,
+  sum(count(*)) over (partition by repo_id order by date_trunc('day', event_date)) as cumul_push_count
+from {{ ref("stg_gharchive") }}
+where event_type = 'Push'
+group by repo_id, repo_name, date_day
```

Comment:
Minor style: I prefer to do `group by 1, 2, 3` to avoid repetition of the column names. That's a subjective preference, I'll leave it up to you to decide :)

Reply:
@edsioufi Updated SQL query; could you please review it?